packages/radix-icons/package.json

```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "description": "Radix Icon Set",
   "author": "Lochlan Bunn <lochlan@modulz.app>",
-  "module": "dist/modulzradix-icons.es.production.js",
+  "module": "dist/radix-icons.esm.js",
   "main": "index.js",
   "typings": "dist/index.d.ts",
```

Comment:
unrelated by shouldnt this be called `"types"`?

Reply:
Yeah sure